eng/versioning/external_dependencies.txt

```diff
@@ -114,73 +114,71 @@ org.messaginghub:pooled-jms;3.0.0
+io.lettuce:lettuce-core;6.2.1.RELEASE
 org.redisson:redisson;3.17.0
-net.bytebuddy:byte-buddy;1.12.17
+net.bytebuddy:byte-buddy;1.12.18
 
 ## Spring boot dependency versions
-org.springframework.boot:spring-boot-dependencies;3.0.0-M5
-org.springframework.boot:spring-boot-actuator;3.0.0-M5
-org.springframework.boot:spring-boot-actuator-autoconfigure;3.0.0-M5
-org.springframework.boot:spring-boot-autoconfigure-processor;3.0.0-M5
-org.springframework.boot:spring-boot-autoconfigure;3.0.0-M5
-org.springframework.boot:spring-boot-configuration-metadata;3.0.0-M5
-org.springframework.boot:spring-boot-configuration-processor;3.0.0-M5
+org.springframework.boot:spring-boot-dependencies;3.0.0-RC1
+org.springframework.boot:spring-boot-actuator;3.0.0-RC1
+org.springframework.boot:spring-boot-actuator-autoconfigure;3.0.0-RC1
+org.springframework.boot:spring-boot-autoconfigure-processor;3.0.0-RC1
+org.springframework.boot:spring-boot-autoconfigure;3.0.0-RC1
+org.springframework.boot:spring-boot-configuration-metadata;3.0.0-RC1
+org.springframework.boot:spring-boot-configuration-processor;3.0.0-RC1
 org.springframework.boot:spring-boot-maven-plugin;3.0.0-M5
-org.springframework.boot:spring-boot-starter-actuator;3.0.0-M5
-org.springframework.boot:spring-boot-starter-aop;3.0.0-M5
-org.springframework.boot:spring-boot-starter-cache;3.0.0-M5
-org.springframework.boot:spring-boot-starter-data-redis;3.0.0-M5
-org.springframework.boot:spring-boot-starter-integration;3.0.0-M5
-org.springframework.boot:spring-boot-starter-logging;3.0.0-M5
-org.springframework.boot:spring-boot-starter-test;3.0.0-M5
-org.springframework.boot:spring-boot-starter-validation;3.0.0-M5
-org.springframework.boot:spring-boot-starter-web;3.0.0-M5
-org.springframework.boot:spring-boot-starter;3.0.0-M5
-org.springframework.boot:spring-boot-test;3.0.0-M5
-org.springframework.boot:spring-boot;3.0.0-M5
-org.springframework.data:spring-data-commons;3.0.0-M6
-org.springframework.data:spring-data-mongodb;4.0.0-M6
-org.springframework.data:spring-data-redis;3.0.0-M6
+org.springframework.boot:spring-boot-starter-actuator;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-aop;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-cache;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-data-redis;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-integration;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-logging;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-test;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-validation;3.0.0-RC1
+org.springframework.boot:spring-boot-starter-web;3.0.0-RC1
+org.springframework.boot:spring-boot-starter;3.0.0-RC1
+org.springframework.boot:spring-boot-test;3.0.0-RC1
+org.springframework.boot:spring-boot;3.0.0-RC1
+org.springframework.data:spring-data-commons;3.0.0-RC1
+org.springframework.data:spring-data-mongodb;4.0.0-RC1
+org.springframework.data:spring-data-redis;3.0.0-RC1
 org.springframework.experimental:spring-aot;0.11.4
```

Comment:
Should we remove this now? Or leave it until we start working on the aot?

Reply:
It should be removed until we start developing the new AOT support, the current native configuration module will use this dependency.